meson.build

```diff
@@ -0,0 +1,52 @@
+# Meson is intended as the future build system of the GnoTime project. For now
+# this file serves for IDE integration only and does nothing more than just
+# building the application.
+#
+# If a Python interpreter supporting Python >= 3.7 is being installed on Ubuntu
+# Trusty Tahr Meson can be used there too for building the application.
+
+project(
+  'GnoTime',
+  'c',
+  license: 'GPL-2.0-or-later',
+  meson_version: '>= 1.0.0',
```

Comment:
This configures fine using Meson 0.49.0, dropping the minimum version to that would enable e.g. building on Debian buster (oldstable).

Seems like you already do support it, minus the version guard. :)

Reply:
I lowered the required Meson version to `0.49.0`. I really don't know Meson well, so when I couldn't `meson compile` with `0.49.0` (generally we are aiming for compatibility with Debian oldstable) I went with `1.0.0` to be on the safe side.